lib/rules/array-bracket-newline.js

```diff
@@ -0,0 +1,232 @@
+/**
+ * @fileoverview Rule to enforce linebreaks after open and before close array brackets
+ * @author Jan Peer Stöcklmair <https://github.com/JPeer264>
+ */
+
+"use strict";
+
+const astUtils = require("../ast-utils");
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+    meta: {
+        docs: {
+            description: "enforce linebreaks after open and before close array brackets",
```

Comment:
Make sure to change the message here to "opening and closing array brackets" like you did with the other messages.